src/app/components/navigation/Footer.tsx

```diff
@@ -4,40 +4,48 @@ import { ListGroupFooter } from "../elements/list-groups/ListGroupFooter";
 import { ListGroupSocial } from "../elements/list-groups/ListGroupSocial";
 import { ListGroupFooterBottom } from "../elements/list-groups/ListGroupFooterBottom";
 import { Link } from "react-router-dom";
+import { ExternalLink } from "../elements/ExternalLink";
 
 export const Footer = () => (
   <footer>
     <div className="footerTop d-print-none">
       <Container>
-        <Row className="px-3 px-sm-0 pb-3 pb-md-4">
-          <Col md="4" lg="3" className="pt-5 logo-col">
-            <div className="d-flex flex-row">
-              <Link to="/">
-                <img src="/assets/logo-mustard.svg" className="footerLogo" alt="Isaac Computer Science homepage" />
-              </Link>
-            </div>
-            <div className="footer-links logo-text pt-3">
+        <Row className="px-3 px-sm-0 pb-3 pb-md-4 footer-content">
```

Comment:
finally, can we check whether we need the `px-3` here, it's adding more padding than needed horizontally to the whole space compared to the designs :)